tests/testHonorCode.sql

```diff
@@ -0,0 +1,21 @@
+-- BLOCK select_exam1
+SELECT
+    a.id
+FROM
+    assessments AS a
+    JOIN assessment_sets AS aset ON (aset.id = a.assessment_set_id)
+WHERE
+    a.course_instance_id = 1
+    AND aset.abbreviation = 'E'
+    AND a.number = '1';
```

Comment:
```suggestion
    AND a.number = $number;
```

Can parameterize this, and delete the `select_exam2` block.